Lib/test/test_pyclbr.py

```diff
@@ -247,7 +247,7 @@ def test_dotted_name_not_a_package(self):
         # not a package.
         #
         # Issue #14798.
-        self.assertRaises(ImportError, pyclbr.readmodule_ex, 'asyncore.foo')
+        self.assertRaises(ImportError, pyclbr.readmodule_ex, 'foo.bar')
```

Comment:
```suggestion
        self.assertRaises(ImportError, pyclbr.readmodule_ex, 'asyncio.foo')
```

Reply:
Thanks @aeros, I think I got confused by the comment in the test which talks about the first part of a dotted name not being a package - got mixed up between packages and modules!